### PR TITLE
Camera calib

### PR DESCRIPTION
**Pull request by DSsoto**
* Calibrated cameras well enough to get a somewhat decent stereo point cloud.
* Enabled down camera launch

image

### CODE REVIEW

File: command/sub8_launch/launch/subsystems/cameras.launch [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <launch>
   <include file="$(find sub8_launch)/launch/subsystems/cameras/launch/front_cameras.launch" />
-  <!-- include file="$(find sub8_launch)/launch/subsystems/cameras/launch/down_cameras.launch" / -->
+  <include file="$(find sub8_launch)/launch/subsystems/cameras/launch/down_cameras.launch" />
```
Comment (kev-the-dev) [inline command/sub8_launch/launch/subsystems/cameras.launch:3]: This probably shouldn't be enabled without a calibration file. At testing today the rectified image topics did not appear

**Review by kev-the-dev — CHANGES_REQUESTED** [review]
Let's not enable down camera in main launch file until it is calibrated. You can recomment it or add the calibration yourself

File: command/sub8_launch/launch/subsystems/cameras.launch (hunk 1 above)
Comment (DSsoto) [reply command/sub8_launch/launch/subsystems/cameras.launch:3]: Nothing bad will happen, we just don't get those topics until we set the calibration.

Comment (DSsoto) [reply command/sub8_launch/launch/subsystems/cameras.launch:3]: I removed the rectification stuff for down camera.

**Review by kev-the-dev — APPROVED** [review]

**Pull request merged**